### ISSUE TITLE
距离计算问题

### ISSUE THREAD

**Issue by LilySys** [michuanhaohao/reid-strong-baseline#106]
罗博士，您好，对于距离计算我存在以下疑惑，恳请您能答疑解惑，谢谢！
疑问一：
reid_metric.py的compute函数中，一开始feats = torch.nn.functional.normalize(feats, dim=1, p=2)已对向量进行了归一化，后面这两行代码求出的是欧式距离呀，为什么你的readme.md中的test说明中，如果TEST.FEAT_NORM "('yes')"为余弦距离？
distmat = torch.pow(qf, 2).sum(dim=1, keepdim=True).expand(m, n) + \
                  torch.pow(gf, 2).sum(dim=1, keepdim=True).expand(n, m).t()
distmat.addmm_(1, -2, qf, gf.t())
疑问二：
使用feats = torch.nn.functional.normalize(feats, dim=1, p=2)对向量进行归一化，如果求余弦距离，qf.gf.t()不就是余弦距离了吗？

**Comment by LilySys** [comment]
第二个疑问我想表达的意思，您代码里面似乎没有实现余弦距离。

**Comment by MolunYu** [comment]
我也有相同的疑惑，请问您解决了吗？

**Comment by MolunYu** [comment]
我也有相同的疑惑，请问您解决了吗？ @LilySys
